utils.go

```diff
@@ -36,6 +36,12 @@ func setupRuntime() (cmd *flag.Command) {
 		handleError(err)
 	}
 
+	if customCfg := cmd.StringFrom("config"); customCfg != "" {
+		fmt.Println("CustomConfig: " + customCfg)
+		configLocation = customCfg
+		cfg, cfgErr = service.NewConfig(configLocation)
```

Comment:
Can we handle this `cfgErr` just in case?

Reply:
we do, but not immediately (in case someone is using a command that does not require a config

Reply:
it's handled on line 59 :)